dev/bots/analyze_sample_code.dart

```diff
@@ -61,7 +78,20 @@ void main(List<String> arguments) {
     flutterPackage = Directory(_defaultFlutterPackage);
   }
 
-  Directory tempDirectory;
+  final bool includeDartUi = parsedArguments.wasParsed('dart-ui-location') || parsedArguments['include-dart-ui'] as bool;
+  late Directory dartUiLocation;
+  if (((parsedArguments['dart-ui-location'] ?? '') as String).isNotEmpty) {
+    dartUiLocation = Directory(
+        path.absolute(parsedArguments['dart-ui-location'] as String));
+    if (!dartUiLocation.existsSync()) {
```

Comment:
The exists check should probably also happen when the defaultDartUiLocation is used (else branch)?

Reply:
Yes, good point. I moved it after the conditional so it'll check both cases.